pkg/controllers/teammembership/util.go

```diff
@@ -0,0 +1,8 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package teammembership
+
+import "time"
+
+const defaultRequeueInterval = 10 * time.Minute
```

Comment:
Please move this directly into the controller, no need to have an extra `util.go` for the constant.